code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -127,6 +129,7 @@
 /datum/chemical_reaction/slime/slimemobspawn/lesser/summon_mobs(datum/reagents/holder, turf/T)
```

Comment:
A few lines above, we attempt to call this and now we're unable to.

Reply:
pardon.